[2025-07-01T23:37:44.535+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-01T23:37:44.558+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-07-01T23:37:26.580282+00:00 [queued]>
[2025-07-01T23:37:44.568+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-07-01T23:37:26.580282+00:00 [queued]>
[2025-07-01T23:37:44.568+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-07-01T23:37:44.585+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-07-01 23:37:26.580282+00:00
[2025-07-01T23:37:44.594+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=196) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-01T23:37:44.596+0000] {standard_task_runner.py:72} INFO - Started process 198 to run task
[2025-07-01T23:37:44.598+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'manual__2025-07-01T23:37:26.580282+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmppnqus81a']
[2025-07-01T23:37:44.599+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask run_dbt_trusted
[2025-07-01T23:37:44.667+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-07-01T23:37:26.580282+00:00 [running]> on host 58ebfa276d50
[2025-07-01T23:37:44.771+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-07-01T23:37:26.580282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-01T23:37:26.580282+00:00'
[2025-07-01T23:37:44.772+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-01T23:37:44.796+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-01T23:37:44.797+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-07-01T23:37:44.811+0000] {subprocess.py:86} INFO - Output:
[2025-07-01T23:37:44.814+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-07-01T23:37:44.815+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-07-01T23:37:48.176+0000] {subprocess.py:93} INFO - [0m23:37:48  Running with dbt=1.8.0
[2025-07-01T23:37:48.543+0000] {subprocess.py:93} INFO - [0m23:37:48  Registered adapter: postgres=1.8.2
[2025-07-01T23:37:49.171+0000] {subprocess.py:93} INFO - [0m23:37:49  Found 9 models, 7 sources, 428 macros
[2025-07-01T23:37:49.176+0000] {subprocess.py:93} INFO - [0m23:37:49
[2025-07-01T23:37:49.380+0000] {subprocess.py:93} INFO - [0m23:37:49  Concurrency: 1 threads (target='dev')
[2025-07-01T23:37:49.381+0000] {subprocess.py:93} INFO - [0m23:37:49
[2025-07-01T23:37:49.399+0000] {subprocess.py:93} INFO - [0m23:37:49  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-07-01T23:37:49.569+0000] {subprocess.py:93} INFO - [0m23:37:49  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 0[0m in 0.17s]
[2025-07-01T23:37:49.573+0000] {subprocess.py:93} INFO - [0m23:37:49  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-07-01T23:37:49.623+0000] {subprocess.py:93} INFO - [0m23:37:49  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 0[0m in 0.05s]
[2025-07-01T23:37:49.626+0000] {subprocess.py:93} INFO - [0m23:37:49  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-07-01T23:37:49.723+0000] {subprocess.py:93} INFO - [0m23:37:49  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 0[0m in 0.09s]
[2025-07-01T23:37:49.731+0000] {subprocess.py:93} INFO - [0m23:37:49  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-07-01T23:37:49.812+0000] {subprocess.py:93} INFO - [0m23:37:49  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 0[0m in 0.08s]
[2025-07-01T23:37:49.833+0000] {subprocess.py:93} INFO - [0m23:37:49
[2025-07-01T23:37:49.834+0000] {subprocess.py:93} INFO - [0m23:37:49  Finished running 4 table models in 0 hours 0 minutes and 0.66 seconds (0.66s).
[2025-07-01T23:37:49.902+0000] {subprocess.py:93} INFO - [0m23:37:49
[2025-07-01T23:37:49.903+0000] {subprocess.py:93} INFO - [0m23:37:49  [32mCompleted successfully[0m
[2025-07-01T23:37:49.904+0000] {subprocess.py:93} INFO - [0m23:37:49
[2025-07-01T23:37:49.905+0000] {subprocess.py:93} INFO - [0m23:37:49  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-07-01T23:37:50.676+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-01T23:37:50.714+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-01T23:37:50.715+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=manual__2025-07-01T23:37:26.580282+00:00, execution_date=20250701T233726, start_date=20250701T233744, end_date=20250701T233750
[2025-07-01T23:37:50.751+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-01T23:37:50.769+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-01T23:37:50.770+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
